DEPRECATIONS.md

```diff
@@ -4,15 +4,15 @@ This document lists deprecated constructs in the SDK, with their expected time 
 
 | API / Feature | Deprecated (warning) | Deprecated (error) | Removed | Notes |
 | --- | --- | --- | --- | --- |
-| Old serialization implementation<br/>*client* | 2021.05.10 | 2021.07.10 ⌛ | 2021.09.10 ⌛ | See the [4.10.0 release notes](https://github.com/GetStream/stream-chat-android/releases/tag/4.10.1) for details |
+| Old serialization implementation<br/>*client* | 2021.05.10 | 2021.07.10 ⌛ | 2021.09.10 ⌛ | See the [4.10.0 release notes](https://github.com/GetStream/stream-chat-android/releases/tag/4.10.0) for details |
```

Comment:
Thanks for catching the wrong release number in the link! Just realized the release version is also missing.

```suggestion
| Old serialization implementation<br/>*client* | 2021.05.10<br/>4.10.0 | 2021.07.10 ⌛ | 2021.09.10 ⌛ | See the [4.10.0 release notes](https://github.com/GetStream/stream-chat-android/releases/tag/4.10.0) for details |
```